### ISSUE TITLE
problems connecting to the db on Ubuntu

### ISSUE THREAD

**Issue by mattlub** [foundersandcoders/express-and-testing-workshop#14]
some ubuntu users not able to connect to the db after following the instructions. (The error was about not having permission to access the db)

one solution was to [add a new superuser with psql and] add `user` and `password` fields to the pg options [here](https://github.com/foundersandcoders/express-and-testing-workshop/blob/master/database/dbConnection.js#L11-L15)

This should be added to the README or the options added to the db connection file to be filled in.

**Comment by mantagen** [comment]
In Gaza, this issue was widespread.
The solution, much like @mattlub has suggested, was to add `user` and `password` fields to the `locaDB` options object. In all/most cases another necessary step was to connect to the postgres CLI, and from inside, run
```
ALTER USER user PASSWORD 'password';
```
This solved all such problems, and the `build:db` script ran swimmingly after that.

**Comment by mattlub** [comment]
came up again: https://github.com/foundersandcoders/nazareth-programme/blob/53756f0b2b688ddaaa21af38f89c24681fa1d718/stop-go-continue/facn3/week8.md#creating-and-testing-express-routes

**Comment by eliascodes** [comment]
I think the better way to resolve this issue is simply to remove the database entirely. There's no need for real persistence in this workshop, the same result can be achieved with a simple in-memory datastore (i.e. just a javascript object), storing the necessary data. Something like:
```js
{
  facsters: {
    firstname: "",
    surname: "",
    cohort: "",
    hobbies: [""],
  },
}
```

Then there are two options:
1. Simply re-write the [queries.js](https://github.com/foundersandcoders/express-and-testing-workshop/blob/master/server/routes/queries.js) file if we keep a `Promise`-based interface
2. Re-write the [queries.js](https://github.com/foundersandcoders/express-and-testing-workshop/blob/master/server/routes/queries.js) file and update the [routes/index.js](https://github.com/foundersandcoders/express-and-testing-workshop/blob/master/server/routes/index.js) file.

Then there's no DB setup step.

**Comment by mattlub** [comment]
yes, I agree, but that is a fair bit more work.

**Comment by eliascodes** [comment]
We'll see if it can be done as part of the curriculum updates in this iteration. Otherwise we can fallback to your solution.

**Comment by jamiecoe** [comment]
@eliascodes I disagree with removing the DB from this exercise. This was an important reference point for FAC-12 when using the `pg-promise`module. Also, a valuable reminder-exercise for creating a new DB on your local machine. 

Would it be possible to refactor `dbConnection.js` so the `localDB` includes `user` and `password` fields? Then we include additional instructions in `Readme`, telling students to add their username and password values to these fields.

Students can check what users they have available with `\du` command, or create a new user with:
`CREATE USER user WITH PASSWORD 'password';`

**Comment by mattlub** [comment]
fixed in #26 :)
